functions/graphql/models/Simulator.js

```diff
@@ -5,14 +5,21 @@ const { firestore } = require("../connectors/firebase");
 // =============================================================================
 
 module.exports = class Simulator {
-  static async createSimulator(name, centerId) {
+  constructor({ id, name, stations, centerId }) {
+    this.id = id;
+    this.name = name;
+    this.centerId = centerId;
+    this.stations = stations;
+  }
+  static async createSimulator(name, stations, centerId) {
     const simulatorData = await firestore()
       .collection("simulators")
       .add({
         name,
+        stations,
```

Comment:
Why has this been added? I know it was added to the GraphQL scheme, but that doesn’t mean it is stored in the database.